cmd/api-linter/cli.go

```diff
@@ -46,7 +46,7 @@ type cli struct {
 	ListRulesFlag           bool
 }
 
-var ExitForLintFailure = errors.New("found problems during linting")
```

Comment:
What was the warning for this? I'd prefer not to unexport this error. Technically it is a breaking change.

Reply:
it just needs a comment, so I'm happy to fix it. 

What part of it is a breaking change? I checked and it wasn't referred to outside of the main package.

Reply:
> What part of it is a breaking change? I checked and it wasn't referred to outside of the main package.

It is exported, so if anyone has depended upon this in their own code, it would break them. I don't really think it would be smart for someone to depend on something in a `cmd/` directory, but it is possible if someone is "scripting" with Go code and handling the errors. If the only problem was missing a comment, let's not even risk annoying someone :)